src/Http/ResponseBuilder.php

```diff
@@ -54,15 +69,19 @@ public function __construct($responseFactory)
      *
      * @param  int|null $statusCode
      * @param  array    $headers
+     * @param  bool     $successFlag
      * @return \Illuminate\Http\JsonResponse
      */
-    public function respond(int $statusCode = null, array $headers = []):JsonResponse
+    public function respond(int $statusCode = null, array $headers = [], bool $successFlag = true):JsonResponse
```

Comment:
I actually removed the public setter as it seemed odd to do `setSuccess(true)` on `error()` and vice versa on `success()`

I think it should be a property that is set explicitly within those methods BUT if someone uses response directly they also need to be able to override the default.

Thoughts?